querqy-for-lucene/querqy-lucene/src/test/java/querqy/lucene/rewrite/LuceneQueryBuilderTest.java

```diff
@@ -362,11 +379,52 @@ public void testMultiMatch01() throws Exception {
         );
     }
 
+    @Test
+    public void testMultiMatch01() throws Exception {
```

Comment:
can we change this to a name that semantically describes what exactly is tested here?